lib/widgets/weekplan_screen_widgets/activity_card.dart

```diff
@@ -97,10 +103,84 @@ class ActivityCard extends StatelessWidget {
     return true;
   }
 
+  ///This function builds the activity card
+  Widget buildChoiceboardAcivityCard(
```

Comment:
should be refactored
```suggestion
  Widget buildChoiceboardActivityCard(
```

Reply:
Done